msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -991,17 +995,18 @@ public static INativeAuthPublicClientApplication createNativeAuthPublicClientApp
      */
     public static INativeAuthPublicClientApplication createNativeAuthPublicClientApplication(
             @NonNull final Context context,
-            @NonNull final NativeAuthPublicClientApplicationConfiguration config) throws MsalException {
+            @NonNull final NativeAuthConfigParameters configParameters) throws MsalException {
```

Comment:
Similarly to other comments
```suggestion
            @NonNull final NativeAuthConfigParameters parameters) throws MsalException {
```

Reply:
Done.